pkg/controller/federation/clusterdeployment_federation_controller.go

```diff
@@ -56,6 +56,9 @@ const (
 	adminKubeconfigKey = "kubeconfig"
 
 	federatedClustersCRDName = "federatedclusters.core.federation.k8s.io"
+
+	clusterDeploymentNamespaceAnnotation = "hive.openshift.io/cluster-deployment-namespace"
+	clusterDeploymentNameAnnotation      = "hive.openshift.io/cluster-deployment-name"
```

Comment:
In other areas i've done namespace/name in an annotation, as it seems a little less cluster on the object but you then have to parse. Do you think splitting into two is a better approach?

Reply:
I guess parsing vs checking if two annotations are set.

Reply:
Makes sense, I switched to one.